docs/examples/types_undefined_warning.py

```diff
@@ -32,19 +32,12 @@ class Person(BaseModel):
 )
 jane_doe = Person(name='Jane Doe', books_read=[python_crash_course])
 
-assert (
-    jane_doe.json(indent=2, exclude_unset=True)
-    == """
-{
-  "name": "Jane Doe",
-  "books_read": [
-    {
-      "title": "Python Crash Course",
-      "author": {
-        "name": "Eric Matthes"
-      }
-    }
-  ]
+assert jane_doe.dict(exclude_unset=True) == {
+    'name': 'Jane Doe',
+    'books_read': [
+        {
+            'title': 'Python Crash Course',
+            'author': {'name': 'Eric Matthes'},
+        },
```

Comment:
As long as there's one book, you won't have an issue. But if there are more books, you may get into an issue with ordering. That's not for now though. I think we're good to solve that problem later, should it present itself.